### PR TITLE
📕 Add bento-fit-text example

### PR DESCRIPTION
**Pull request by sebastianbenz**
(also fix readme formatting error)

### CODE REVIEW

File: extensions/amp-fit-text/1.0/example/index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <div class="buttons" style="margin-top: 8px">
+      <button id="font-button">Change max-font-size</button>
+      <button id="content-button">Change content</button>
+    </div>
+
+    <script>
+      (async () => {
+        const fitText = document.querySelector('#my-fit-text');
+        await customElements.whenDefined('bento-fit-text');
+        console.log('regustered');
```
Comment (kvchari) [inline extensions/amp-fit-text/1.0/example/index.html:53]: typo

**Review by kvchari — APPROVED** [review]
nit: there's a typo, but otherwise looks fine

**Comment by sebastianbenz** [comment]
I don't think the test failures are related to this  PR...

File: extensions/amp-fit-text/1.0/example/index.html (hunk 1 above)
Comment (sebastianbenz) [reply extensions/amp-fit-text/1.0/example/index.html]: Leftover. Removed...

**Pull request merged**